tests/PageObjects/PostGameResultsPage.ts

```diff
@@ -47,7 +51,7 @@ export class PostGameResultsPagePO {
     return this.page.getByTestId('highscores-button');
   }
 
-  public async goNext() {
+  public async goToPostGameHighScoresStep() {
```

Comment:
I think `goToHightScoresStep` would be enough (PostGame is already in the PO name)